main.rb

```diff
@@ -0,0 +1,50 @@
+require './app'
+
+def menu
+  puts 'Please choose an option by entering a number: '
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+  user_choice = gets.chomp.to_i
+  puts user_choice
+  execute(user_choice)
+end
```

Comment:
- [ ] Hello, The App is falling to list the books and people I have created. Can you Fix the issue and submit again for re-review?
- NB: I would advise you checking the APP if it is working 100% before submitting, to save time. 
-
![Capture](https://github.com/patriciachrysy/oop-school-library/assets/63972670/e7277c9a-d729-4956-a4e5-7342c97e9725)

Reply:
It's curious because it is working just fine on my side

Can you check again?
![image](https://github.com/patriciachrysy/oop-school-library/assets/23379326/b115978c-88fb-4d10-82df-51b9984036bc)

Reply:
- Hello, I cross-checked at my end before making this final decision. Let another reviewer check it out. Thank you